go/vt/vtgate/engine/ordered_aggregate.go

```diff
@@ -127,18 +131,23 @@ func (ap *AggregateParams) String() string {
 		collation := collations.Local().LookupByID(ap.CollationID)
 		keyCol += " COLLATE " + collation.Name()
 	}
+	dispOrigOp := ""
+	if ap.OrigOpcode != AggregateUnassigned && ap.OrigOpcode != ap.Opcode {
+		dispOrigOp = "_" + ap.OrigOpcode.String()
+	}
 	if ap.Alias != "" {
-		return fmt.Sprintf("%s(%s) AS %s", ap.Opcode.String(), keyCol, ap.Alias)
+		return fmt.Sprintf("%s%s(%s) AS %s", ap.Opcode.String(), dispOrigOp, keyCol, ap.Alias)
 	}
-	return fmt.Sprintf("%s(%s)", ap.Opcode.String(), keyCol)
+	return fmt.Sprintf("%s%s(%s)", ap.Opcode.String(), dispOrigOp, keyCol)
```

Comment:
I think the plans would be more readable if we had a field that contains the original op code to the description instead of suffixing the original op code to the current op code. I don't have strong opinions though, the current change works well too imo.